src/core/Events.d.ts

```diff
@@ -0,0 +1,177 @@
+import { Class } from "./Class";
+
+export type EventMap = {
+  [name: string]: LeafletEvent;
+};
+
+export interface LeafletEvent {
+  type: string;
+  target: any;
+  sourceTarget: any;
+  propagatedFrom: any;
+  /**
+   * @deprecated The same as {@link LeafletEvent.propagatedFrom propagatedFrom}.
```

Comment:
I personally prefer a clear describing action: `@deprecated Use insteaed {@link LeafletEvent.propagatedFrom propagatedFrom}`

Reply:
Just copied it. I agree with you will change that.